specs/OracleQueries.yml

```diff
@@ -1559,6 +1585,30 @@ components:
           type: string
           description: The region
           example: "Surrey"
+    managingOfficerData:
+      type: object
+      required:
+        - residential_address
+        - contact_name_full
+        - contact_email_address
+        - date_of_birth
+      properties:
+        residential_address:
+          type: object
+          description: The residential address of the managing officer
+          example: "123 Updog road"
+        contact_name_full:
+          type: string
+          description: The full contact name of the managing director
+          example: "John doe"
+        contact_email_address:
+          type: string
+          description: The contact email address of the managing officer
+          example: "testing@email.com"
+        date_of_birth:
+          type: string
+          description: The date of birth of the managing officer
+          example: "20-1-1930"
```

Comment:
I think it's usually year-month-day ("1930-01-20"). Might be better to use that?

Reply:
fixed